runtime/vm/JFRConstantPoolTypes.cpp

```diff
@@ -509,7 +509,7 @@ VM_JFRConstantPoolTypes::addPackageEntry(J9Class *clazz)
 	}
 
 	entry->packageName = (U_8*) packageName;
```

Comment:
The type of the field `packageName` and the local of the same name should be `const U_8 *` so this cast and the one on line 505 would not be needed. The cast on line 1280 is missing the `const` qualifier and that line should be split:
```c
		j9tty_printf(
				PORTLIB,
				"%u) moduleIndex=%u packageName=%.*s exported=%u\n",
				tableEntry->index,
				tableEntry->moduleIndex,
				tableEntry->packageNameLength,
				(const char *)tableEntry->packageName,
				tableEntry->exported);
```